radis/lbl/broadening.py

```diff
@@ -2248,6 +2248,19 @@ def _broaden_lines(self, df):
                 N = int(len(df) * len(wavenumber) / chunksize) + 1
                 # Too big may be faster but overload memory.
                 # See Performance for more information
+
+                # Raise performance warning if the chunks are bigger
+                # than the number of lines.
+                try:
+                    assert N < len(df)
+                except AssertionError:
+                    self.warn(
+                        "Chunksize is currently too small according"
+                        + " to the dataframe size. Please set a larger"
+                        + " value of chunksize.",
```

Comment:
you don't say why, it's not very clear. 
Can you try something like  
```
"There are currently more chunks ({N:.3e}) than lines ({len(df}:.3e), calculation times will be extremelly slow. Try to have at least 10,000 - 100,000 lines per chunk. We suggest increasing chunksize to {10000*len(wavenumber):.1e} - {100000*len(wavenumber):.1e}" 
```
Note : numbers are kinda guesses here. Check it works.

Reply:
Hello! Apologies for not making it descriptive. I tried the above with CO, CO2, N2O and NO molecules, and it works well!